src/components/AddHouseForm/AddHouseForm.tsx

```diff
@@ -24,6 +25,24 @@ import getRandomString from '/src/utils/getRandomString';
 import styles from './AddHouseForm.module.css';
 import FacilityCheckbox from './components/FacilityCheckbox/FacilityCheckbox';
 
+interface FieldsSchema {
+  streetNumber: string;
```

Comment:
Shouldn't there be a streetNumber: number?

Reply:
It should be a string because of street numbers like "21E" :)